.gitignore

```diff
@@ -13,3 +13,6 @@ firebase-debug.log
 .idea
 .DS_Store
 .vscode/
+
+# env
+.env.dev
```

Comment:
This should not be ignored.  Please use .env.dev.local if you need to override settings: https://vitejs.dev/guide/env-and-mode.html#env-files